fe4u-boilerplate-main/src/js/API/API.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+const api = require("./callApi");
```

Comment:
`callAPI.js`:
```
export function callApi(apiURL, endpoint, method) {
```
`API.js`:
```
import { callApi } from './callApi'
```
But the file `callApi.js` usually is called `services.js`. All methods, which make requests to BE, should be placed in `services.js`.